Entity/SnapshotManager.php

```diff
@@ -147,6 +168,8 @@ public function findEnableSnapshot(array $criteria)
      * @param string $routeName
      *
      * @return PageInterface|false
+     *
+     * @deprecated since 3.x, to be removed in 4.0
```

Comment:
Here too, this method is deprecated but nothing on the upgrade doc.

Please @sonata-project/contributors do not merge PR without any note on the upgrade note.